tests/raftstore/test_conf_change.rs

```diff
@@ -337,6 +337,7 @@ fn test_after_remove_itself<T: Simulator>(cluster: &mut Cluster<T>) {
 
     // disable auto compact log.
     cluster.cfg.raft_store.raft_log_gc_threshold = 10000;
+    cluster.cfg.raft_store.allow_remove_leader = true;
```

Comment:
Then many tests need to be guaranteed that leader should not be the target of `must_remove_peer`.